plugins/extractors/presto/presto_test.go

```diff
@@ -35,8 +35,8 @@ var (
 func TestMain(m *testing.M) {
 	// setup test
 	opts := dockertest.RunOptions{
-		Repository:   "ahanaio/prestodb-sandbox",
-		Tag:          "0.270",
+		Repository:   "ghcr.io/popsql/prestodb-sandbox",
+		Tag:          "0.281",
```

Comment:
looking at this https://github.com/popsql/prestodb-sandbox/pkgs/container/prestodb-sandbox latest is 0.286, any reason why we're using 0.281?

Reply:
no particular reason other than anticipating if there's any breaking changes in the latest, let me try to bump to the latest see if it's break